electron/electron.ts

```diff
@@ -41,7 +42,10 @@ const BUILD_DIR = join(__dirname, '..', '..', 'build');
 const IS_TEST = process.env.NODE_ENV === 'test';
 const TEST_PORT = process.env.TEST_PORT;
 
+let mainWindowEmitter: EventEmitter | null = null;
```

Comment:
Instead of making this emitter global, can we return it from `createWindow` and  use it in the `then` closure for `appWhenReady` below?

Reply:
Perfect 🙌